### ISSUE TITLE
[Bug]: CICD Pipeline Permissions error preventing from triggering pipeline. "you do not have permission to update the application" NoPermissionToUpdateApplication

### ISSUE THREAD

**Issue by mokkang** [Checkmarx/ast-cli#1317]
### Contact Details

jmokkang@gmail.com

### What happened?

CICD pipelines using checkmarx/ast-cli:latest (which is using 2.3.36 [bb2cc2dd3f24](https://hub.docker.com/layers/checkmarx/ast-cli/2.3.36/images/sha256-bb2cc2dd3f24bf19f8a02aae8d71dd00bafa3eb13bffeb9e685e34302cb30af4)), 

None of the Gitlab pipelines are being triggered as it is errors out due to this line. 	`NoPermissionToUpdateApplication        = "you do not have permission to update the application" `
from this file. 
https://github.com/Checkmarx/ast-cli/blob/43306672031ba1cd4a64ead66136499eae0b6c09/internal/constants/errors/errors.go#L27

All of my pipelines were working prior to 6am 09/26/2025 PST. 

### Version

2.3.36

### Operating System & Version

Other

### Specify Other OS Version

docker image for ast-cli 

### Relevant log output

```shell
Creating directory
you do not have permission to update the application
Error: you do not have permission to update the application
Cleaning up project direcctory and file based variables.
```

**Comment by mokkang** [comment]
Update Application permission will need to be granted for the Oauth2 and API key to submit scans. please confirm
